src/components/select/utils/get-next-child-by-key.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const homeKey = 'Home';
```

Comment:
These variables are only used in one place. I think having them defined as constants instead of using the strings makes the code more difficult to read.

Reply:
removed